src/components/Gallery/Gallery.scss

```diff
@@ -0,0 +1,13 @@
+
+.gallery {
+  display: flex;
+  justify-content: center;
+  margin: auto;
+  overflow: auto;
+  white-space: nowrap;
+  padding: 10px 0;
+  .gallery__images--image {
+    width: 200px;
+    border-radius: 10px 10px;
+  }
```

Comment:
check the bem on this one feels strange to have gallery__images--image could it not just be
```
gallery__images {
//some css
 &--image{
//some css
}
}
```